Install Python scripts to /usr/share/clfft and not to /usr/bin.

Patch by Marius Brehler.
--- scripts/perf/CMakeLists.txt
+++ scripts/perf/CMakeLists.txt
@@ -21,4 +21,4 @@ set(GRAPHING_SCRIPTS 	measurePerformance
 						performanceUtility.py
 						)
 
-install( FILES ${GRAPHING_SCRIPTS} DESTINATION bin${SUFFIX_BIN} )
\ No newline at end of file
+install( FILES ${GRAPHING_SCRIPTS} DESTINATION share/clfft/${SUFFIX_BIN} )
\ No newline at end of file
